integration-tests/helpers/index.js

```diff
@@ -284,40 +296,44 @@ async function createSandbox (dependencies = [], isGitRepo = false,
  *
  * @param {object} sandbox - A `sandbox` as returned from `createSandbox`
  * @param {string} filename - The file that will be copied and modified for each variant.
- * @param {object} variants - The variants. If empty then a default import style will be added
- * depending on the parameters passed through.
- * @param {object} bindingName - The binding name that will be use to bind to the packageName.
- * @param {object} namedVariant - The name of the named variant to use.
- * @param {object} packageName - The name of the package.
- * @returns {object} A map from variant names to resulting filenames
+ * @param {Variants} variants - The variants.
+ * @returns {Variants} A map from variant names to resulting filenames
  */
-function varySandbox (sandbox, filename, variants, bindingName, namedVariant, packageName) {
+function varySandbox (sandbox, filename, variants, namedVariant, packageName = variants) {
+  if (typeof variants === 'string') {
+    const bindingName = variants
+    variants = {
+      default: `import ${bindingName} from '${packageName}'`,
+      star: namedVariant
+        ? `import * as ${bindingName} from '${packageName}'`
+        : `import * as mod${bindingName} from '${packageName}'; const ${bindingName} = mod${bindingName}.default`,
+      destructure: namedVariant
+        ? `import { ${namedVariant} } from '${packageName}'; const ${bindingName} = { ${namedVariant} }`
+        : `import { default as ${bindingName}} from '${packageName}'`
+    }
+  }
+
   const origFileData = readFileSync(path.join(sandbox.folder, filename), 'utf8')
   const [prefix, suffix] = filename.split('.')
```

Comment:
Nit: I'm aware this hasn't changed since the old implementation, but if `filename` contains multiple periods, this will fail. Not sure if that's ever the case in reality, but doing it like this would be safer:

```diff
-  const [prefix, suffix] = filename.split('.')
+  const { name: prefix, ext: suffix } = path.parse(filename)
   const variantFilenames = /** @type {Variants} */ ({})

   for (const [variant, value] of Object.entries(variants)) {
-    const variantFilename = `${prefix}-${variant}.${suffix}`
+    const variantFilename = `${prefix}-${variant}${suffix}`
```